packaging/vm-deployment/Vagrantfile

```diff
@@ -0,0 +1,15 @@
+# -*- mode: ruby -*-
```

Comment:
Did you try using [this module[(https://docs.ansible.com/ansible/latest/collections/community/libvirt/virt_module.html)
Example: https://www.redhat.com/sysadmin/build-VM-fast-ansible

That would mean we don't need Vagrant :tada: and users on macOS would be OK as I *hope* libvirt supports using the qemu hvf backend - see: [https://libvirt.org/macos.html].

Reply:
I've seen that post before but never tried it. It would require more files (roles, galaxy dir, ansible.cfg, libvirt template etc.) and require a bit more maintenance (OS versioning that vagrant does for you etc). Good either way! Might be worth doing it on another PR since it will take me a few to test it out across various platforms. Ty!